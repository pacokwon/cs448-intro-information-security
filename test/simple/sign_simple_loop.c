#include "homework.h"

int main()
{
    for(int i = 0; i < 100; i++)
    {
        int x = 100 / i;
    }

    return 0;
}
