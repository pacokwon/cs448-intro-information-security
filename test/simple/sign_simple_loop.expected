Potential Division-by-zero @ simple/sign_simple_loop.c:main:7:21, %i.0 = Top
